src/pages/Login/Login.layout.vue

```diff
@@ -1,50 +1,72 @@
 <template>
-  <q-form
-    @submit="() => onSubmit({ email: emailValue, password: passwordValue })"
-    class="q-gutter-md"
-  >
-    <q-input
-      filled
-      :label="emailAttrs.label"
-      :hint="emailAttrs.hint"
-      :type="emailAttrs.type"
-      :model-value="emailValue"
-      @update:model-value="$emit('update:emailValue', $event)"
-      lazy-rules
-      :rules="emailValidations"
-    />
+  <div class="login__container">
+    <header>
+      <img class="login__logo" :src="didlogo" alt="DID Logo" />
```

Comment:
maybe change the name of the image file to "didLogo"

Reply:
I'll do!

Reply:
I miss styled components hahaha